src/startupscriptgenerator/src/node/scriptgenerator.go

```diff
@@ -71,14 +71,15 @@ func createApplicationInsightsLoaderFile(appInsightsLoaderFilePath string) {
 		var prefixInternalSdkVersion = function (envelope, contextObjects) {
 			try {
 				var appInsightsSDKVersion = appInsights.defaultClient.context.keys.internalSdkVersion;
-				envelope.tags[appInsightsSDKVersion] = "ad_" + envelope.tags[appInsightsSDKVersion];
+				envelope.tags[appInsightsSDKVersion] = "alr_" + envelope.tags[appInsightsSDKVersion];
```

Comment:
We seems to skip creating this loader file if there is one already on disk (it could have been created in previous release) and might be incompatible with these new changes(?). We should recreate this file. cc @arroyc

Reply:
IMO, it will be better if appinsights provides us this file  with the sdk, we would then update the path in the script. what do you think @kichalla @markwolff.

Reply:
Hmm, that is a good idea actually. @markwolff As Arjun mentioned here, if the loader file is going to be a standard one which does not have any other external dependencies it could be part of the applicationinsights module itself?

Reply:
Seems good. An issue is that eventually the loader script would contain each compute platform's internal telemetry & self-troubleshooting logic. Maybe the loader abstracts it out and lets it be set by each platform before executing the rest of the loader.

Reply:
@markwolff, in future - this should be part of your agent. in java case we use special marker file in order to determine the prefix